src/GregClient/Requests/GetMaintainers.cs

```diff
@@ -0,0 +1,32 @@
+﻿using RestSharp;
+
+namespace Greg.Requests
+{
+    public class GetMaintainers : PackageReferenceRequest
+    {
+
+        public GetMaintainers(string engine, string name)
+        {
+            this._name = name;
+            this._engine = engine;
+        }
+
+        public override string Path
+        {
+            get
+            {
+                return "package/" + this._engine + "/" + this._name + "/maintainers";
+            }
+        }
+
+        public override Method HttpMethod
+        {
+            get { return Method.GET; }
+        }
+
+        internal override void Build(ref RestRequest request)
```

Comment:
why does this need an empty implementation?

Reply:
It is an abstract method overridden from Request class that needs to be implemented here.

Reply:
`GetMaintainers` class inherit `PackageReferenceRequest` which further inherits `Request` class that has `Build` abstract method.

Reply:
is the reason it is empty that it is a get request?